readme.md

```diff
@@ -53,7 +53,71 @@ Directory names should be lowercase. Filenames should be singular.
 *todo: evaluate common prefix like `pagelib` and BEM naming.*
 
 ## Development setup and workflow
-*Coming soon.*
+The Android and iOS Wikipedia apps make use of the `wikimedia-page-library`. At some point their respective build phases invoke `npm install` which causes a *published* version of this library to be retrieved and used in their respective builds. 
+
+But sometimes we want to test *unpublished* `wikimedia-page-library` branches. This could be to review bug fixes or new feature branches or to develop fixes and features ourselves. We'd like to be able to checkout a local branch of `wikimedia-page-library` and run the app's build process and have the app use whatever we had checked out of our local copy of `wikimedia-page-library`.
+
+To do this:
+
+**Step 1** (only need to do this once)
+
+Create a local clone of `wikimedia-page-library`:
+> `git clone https://github.com/wikimedia/wikimedia-page-library.git`
+
+**Step 2** (do this as needed)
+
+Checkout the branch to be tested:
+> `cd ~/wikimedia-page-library/`
```

Comment:
Consider dropping the absolute path reference here and let the user figure out where they've cloned the repo, so `cd wikimedia-page-library`.